src/AppInstallerCLIE2ETests/Test.runsettings

```diff
@@ -14,13 +14,19 @@
        LooseFileRegistration: Bool to set if loose file registration should be used.
        InvokeCommandInDesktopPackage: Bool to indicate using Invoke-CommandInDesktopPackage for test execution.
                                       This is used when AppExecutionAlias is not available, or disabled.
+       StaticFileRootPath: Path to the set of static test files that will be served as the source for testing purposes.
+	   ExeTestInstallerPath: The Exe Installer executable under test.
+	   PackageCertificatePath: Signing Certificate Path used to certify Index Source Package
   -->
   <TestRunParameters>
-    <Parameter name="PackagedContext" value="true" />
-    <Parameter name="VerboseLogging" value="false" />
-    <Parameter name="AICLIPath" value="AppInst.exe" />
-    <Parameter name="AICLIPackagePath" value="AppInstallerCLIPackage.appxbundle" />
-    <Parameter name="LooseFileRegistration" value="false" />
-    <Parameter name="InvokeCommandInDesktopPackage" value="false" />
+	  <Parameter name="PackagedContext" value="false" />
+	  <Parameter name="VerboseLogging" value="false" />
+	  <Parameter name="AICLIPath" value="C:\Users\ryfu\source\repos\winget-cli\src\x64\Debug\AppInstallerCLI\AppInstallerCLI.exe" />

```

Comment:
>C:\Users\ryfu\source\repos\winget-cli\src\x64\Debug\AppInstallerCLI\AppInstallerCLI.exe [](start = 38, length = 87)

These need to be some default value that indicates a required override (as do others below).

Reply:
Reverted back to Default values